client/keys.go

```diff
@@ -104,6 +104,9 @@ type KeysAPI interface {
 	// Get retrieves a set of Nodes from etcd
 	Get(ctx context.Context, key string, opts *GetOptions) (*Response, error)
 
+	// Exist wraps Get method returning true when the key exists
+	Exist(ctx context.Context, key string, opts *ExistOptions) (bool, error)
```

Comment:
I do not feel we need this method. I feel expose `IsKeyNotFound` in util.go is good enough.